### PR TITLE
I added 3 new malicious links

### PR DESCRIPTION
**Pull request by muratbz**
Fake instagram links targeting users from Turkey

### CODE REVIEW

File: add-link [hunk 1]
```diff
@@ -38,4 +38,6 @@ https://www.mdscbenin.org/wp-includes/theme-compat/dar/newsant1/login.php
 https://nort2on.com/setup/
 http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
-
+https://ig-feedbackcenter.com/form/983462456879/
+http://veryfincopyrihts.com/
```
Comment (spirillen) [inline add-link:42]: Can't confirm this one for `bitninja.io` are blocking for the access.

**Review by spirillen — COMMENTED** [review]
Needs more information for `veryfincopyrihts.com` as I can't confirm it is phishing

**Comment by muratbz** [comment]
> Needs more information for `veryfincopyrihts.com` as I can't confirm it is phishing

Other links?

**Comment by muratbz** [comment]
> Needs more information for `veryfincopyrihts.com` as I can't confirm it is phishing

http://veryfincopyrihts.com/contact/

**Comment by spirillen** [comment]
I get a 404 on that site now, can you confirm this? And if, please remove the line from the sibmission so we can have the other two phish blocked ASAP

image

**Comment by muratbz** [comment]
> I get a 404 on that site now, can you confirm this? And if, please remove the line from the sibmission so we can have the other two phish blocked ASAP
> 
> image

yup I contacted the domain name provider and complained. I'm leaving them as the others are on different domain provider. I removed the dead link.

**Review by muratbz — COMMENTED** [review]
Dead link removed

**Comment by muratbz** [comment]
Dude, you're too slow.

**Pull request closed without merging**